charts/databend-query/values.yaml

```diff
@@ -93,6 +93,14 @@ config:
 
   # [meta]
   meta:
+    # If databend-meta is hosted in the same cluster as databend-data, you can enable this to generate endpoints in a K8s native way.
+    generateEndpoints: false
+    # Databend-meta replica count
+    metaReplicas: 3
```

Comment:
Redundant `meta` prefix?

Reply:
Yes, can be removed. Was trying to clarify that this is not related to the query set replicas.